test/DebugInfo/DebugTypeMember.ll

```diff
@@ -0,0 +1,64 @@
+; Exercise Value field in SPIRV TypeMember.
+; It is used to preserve DIDerivedType's extraData argument in
+; LLVM IR when a DIFlagStaticMember is present.
+
+; Original *.cpp source
+; 
+; namespace {
+; struct anon_static_decl_struct {
+;   static const int anon_static_decl_var = 117 + 234;
+; };
+; }
+; int ref() {
+;   return anon_static_decl_struct::anon_static_decl_var;
+; }
+
+; RUN: llvm-as %s -o %t.bc
+
+; RUN: llvm-spirv -o %t.spv %t.bc
+; RUN: llvm-spirv -r -o %t.rev.bc %t.spv
+; RUN: llvm-dis %t.rev.bc -o %t.rev.ll
+; RUN: FileCheck %s --input-file %t.rev.ll
+
+; RUN: llvm-spirv -o %t.ns.spv %t.bc --spirv-debug-info-version=nonsemantic-shader-200
```

Comment:
we should probably extend testing with --spirv-debug-info-version=nonsemantic-shader-100
and by adding few SPIR-V checks, other then that - LGTM